src/diffusers/loaders/transformer_flux.py

```diff
@@ -177,5 +177,3 @@ def _load_ip_adapter_weights(self, state_dicts, low_cpu_mem_usage=False):
 
         self.encoder_hid_proj = MultiIPAdapterImageProjection(image_projection_layers)
         self.config.encoder_hid_dim_type = "ip_image_proj"
-
```

Comment:
let's do this?
```python
self.encoder_hid_proj.to(dtype=self.dtype, device=self.device)
```

if this works, can we apply this to other ip-adapter loader as well?

Reply:
I don't think we need to cast it?

Reply:
oh I think you're right!